### PR TITLE
[merged] Start/stop syscontainers with atomic run/stop

### PR DESCRIPTION
**Pull request by yuqi-zhang**
Wrap atomic run/stop with systemctl start/stop for system
containers. This way the user can directly use the atomic
CLI for the full container cycle. In addition, the equivalent
functionality already exists for docker containers, so it
makes sense to allow system containers to start/stop in a
similar fashion.

Signed-off-by: Yu Qi Zhang <jerzhang@redhat.com>

### CODE REVIEW

File: Atomic/syscontainers.py [hunk 1]
```diff
@@ -677,6 +677,18 @@ def _is_service_failed(self, name):
             else:
                 return False
 
+    def start_service(self, name):
+        try:
+            self._systemctl_command("start", name)
+        except subprocess.CalledProcessError as e:
+            util.write_out(e.output)
```
Comment (rhatdan) [inline Atomic/syscontainers.py]: If this is an error you should raise ValueError(e.output)

File: Atomic/syscontainers.py [hunk 2]
```diff
@@ -677,6 +677,18 @@ def _is_service_failed(self, name):
...
+
+    def stop_service(self, name):
+        try:
+            self._systemctl_command("stop", name)
+        except subprocess.CalledProcessError as e:
+            util.write_out(e.output)
```
Comment (rhatdan) [inline Atomic/syscontainers.py]: If this is an error you should raise ValueError(e.output)

**Comment by yuqi-zhang** [comment]
Will fix the error

File: Atomic/syscontainers.py (hunk 2 above)
Comment (yuqi-zhang) [inline Atomic/syscontainers.py]: Changed, thanks!

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**